This shouldn't be necessary as it's redundant with the other patch
But some SD cards need that fix too
---
diff --git a/include/linux/mmc/card.h b/include/linux/mmc/card.h
index 279b39008a33..b15af30d394c 100644
--- a/include/linux/mmc/card.h
+++ b/include/linux/mmc/card.h
@@ -151,7 +151,9 @@ struct sd_switch_caps {
 	unsigned int		hs_max_dtr;
 	unsigned int		uhs_max_dtr;
 #define HIGH_SPEED_MAX_DTR	50000000
-#define UHS_SDR104_MAX_DTR	208000000
+//#define UHS_SDR104_MAX_DTR	208000000
+// Blame faulty switch's SDHCI bus
+#define UHS_SDR104_MAX_DTR	100000000
 #define UHS_SDR50_MAX_DTR	100000000
 #define UHS_DDR50_MAX_DTR	50000000
 #define UHS_SDR25_MAX_DTR	UHS_DDR50_MAX_DTR
